include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort_one_wg.h

```diff
@@ -166,16 +165,12 @@ struct __subgroup_radix_sort
 
                 __cgh.parallel_for<_Name...>(
                     __range, ([=](sycl::nd_item<1> __it)[[_ONEDPL_SYCL_REQD_SUB_GROUP_SIZE(__req_sub_group_size)]] {
-                        _KeyT __keys[__block_size];
+                        _ValT __keys[__block_size];
```

Comment:
Also, I'd recommend renaming this variable `__keys` to `__values` or something similar to match the type changes.

Reply:
good point... with C++ perspective, yes, it requires  default ctor for _ValT..
So, to be accuracy we have to add constructor calls instead of copying, movements and destructor calls..

Reply:
> good point... with C++ perspective, yes, it requires default ctor for _ValT.. So, to be accuracy we have to add constructor calls, movements and destructor calls..

Please somebody to review the new introduced changes regarding ctor/moving/destructor.